dotnet-engine/Yggdrasil.Engine/Yggdrasil.Engine.csproj

```diff
@@ -6,8 +6,8 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <PackageId>Unleash.Yggdrasil</PackageId>
-    <Version>1.0.4</Version>
-    <YggdrasilCoreVersion>0.14.2</YggdrasilCoreVersion>
+    <Version>1.0.5-beta.1</Version>
```

Comment:
Are we bumping this when merging? appears it's already been sent up to NuGet.. 🤷

Reply:
someone's not paying attention - disregard!